anndata/core/aligned_mapping.py

```diff
@@ -90,30 +114,30 @@ def axes(self) -> Tuple[int, ...]:
 
     @property
     @abstractmethod
-    def isview(self) -> bool:
+    def is_view(self) -> bool:
         pass
 
     @property
-    def parent(self) -> "AnnData":
+    def parent(self) -> Union["anndata.AnnData", "raw.Raw"]:
         return self._parent
 
-    def copy(self):
-        d = self._actual_class(self.parent, self._axis)
+    def copy(self) -> "_Actual":
```

Comment:
It is now!

Reply:
Ah, sorry, I misunderstood. This is on the base class, therefore this returns an instance of the class variable (which is a subclass of AlignedActual). So this is correct.

Reply:
Can you point me to the docs for that?

Reply:
You’re right, it doesn’t work: python/mypy#8070

Reply:
I think `-> AlignedView` would be fine for now.

Otherwise, not totally sure off the top of my head. Maybe something like

```python
from typing import Protocol, TypeVar

class View():
    pass

class Actual():
    pass

VT = TypeVar("VT", View)
AT = TypeVar("AT", Actual)

class AlignedMapping(Protocol[AT, VT]):
    _Actual: AT
    _View: VT

    def copy(self) -> AT:
        pass

class Layers("AlignedMapping[_LayersActual, _LayersView]"):
    pass
```